coding-tasks/251204-secret-santa-assigner/assign_santa.py

```
@@ -0,0 +1,275 @@
+        list of names as strings
+
+    Raises
+    ------
+    ValueError
+        IF there is no 'name' column in the input names file or less than 2
+        participants given as input
+    """
+    if "name" not in input_names_df:
+        raise ValueError(
+            "Input names file does not have required 'name' column header"
+        )
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)